Basic_viewer/doc/Basic_viewer/CGAL/Graphic_storage.h

```diff
@@ -0,0 +1,87 @@
+
+namespace CGAL {
+
+/*!
+ \ingroup PkgBasicViewerClasses
+
+The class `Graphic_storage` stores points, segments, triangles, rays and lines. Elements can be added, possibly with associated colors. Non triangular faces can be directly added and are triangulated internally.
```

Comment:
Where does the name `Graphic_storage` come from.  What does it correspond to in other graphics related APIs. Just googling this word I do not find anything that corresponds.

Reply:
From my mind ;)
This class allows to store points, segments and triangles. It can be used for drawing purpose, but also maybe for other options.

Reply:
Does this correspond to the   `QGraphicsScene` in the Qt GraphicsView framework?

Reply:
More or less; but QGraphicsScene has much more functionalities.

Reply:
They have this in common: 
 "Note that QGraphicsScene has no visual appearance of its own; it only manages the items. You need to create a [QGraphicsView](https://doc.qt.io/qt-6/qgraphicsview.html) widget to visualize the scene."
So I would rename `Graphic_storage` to `Graphics_scene".

Reply:
ok why not.

Reply:
ok also by @sloriot

Reply:
done